addons/io_scene_gltf2/__init__.py

```diff
@@ -429,6 +443,19 @@ def execute(self, context):
         export_settings['gltf_binaryfilename'] = os.path.splitext(os.path.basename(
             bpy.path.ensure_ext(self.filepath,self.filename_ext)))[0] + '.bin'
 
+        user_extensions = []
+
+        if bpy.app.version < (2,80,0):
+            pass
```

Comment:
Reason not to do 2.79?

Reply:
Need to check how to do the UI stuff. If posible, I prefer to get it merged and later we can work on the 2.79 version if needed

Reply:
Ok. You still need to update invoke for it though.